app/default-files/default-themes/simple/partials/footer.hbs

```diff
@@ -3,48 +3,55 @@
         <div class="footer__social">
             {{#if @config.custom.socialFacebook}}
                 <a href="{{@config.custom.socialFacebook}}" aria-label="Facebook">
+                    <desc>Facebook</desc>
                     <svg>
                         <use xlink:href="{{@website.assetsUrl}}/svg/svg-map.svg#facebook"/>
                     </svg>
                 </a>
             {{/if}}
             {{#if @config.custom.socialTwitter}}
                 <a href="{{@config.custom.socialTwitter}}" aria-label="Twitter">
+                    <desc>Twitter</desc>
                     <svg>
                         <use xlink:href="{{@website.assetsUrl}}/svg/svg-map.svg#twitter"/>
                     </svg>
                 </a>
             {{/if}}
             {{#if @config.custom.socialInstagram}}
                 <a href="{{@config.custom.socialInstagram}}" aria-label="Instagram">
+                    <desc>Instagram</desc>
                     <svg>
                         <use xlink:href="{{@website.assetsUrl}}/svg/svg-map.svg#instagram"/>
                     </svg>
                 </a>
             {{/if}}
             {{#if @config.custom.socialLinkedin}}
                 <a href="{{@config.custom.socialLinkedin}}" aria-label="LinkedIn">
+                    <desc>LinkedIn</desc>
                     <svg>
                         <use xlink:href="{{@website.assetsUrl}}/svg/svg-map.svg#linkedin"/>
                     </svg>
                 </a>
             {{/if}}
             {{#if @config.custom.socialVimeo}}
                 <a href="{{@config.custom.socialVimeo}}" aria-label="Vimeo">
+                    <desc>Vimeo</desc>
                     <svg>
                         <use xlink:href="{{@website.assetsUrl}}/svg/svg-map.svg#vimeo"/>
                     </svg>
                 </a>
             {{/if}}
             {{#if @config.custom.socialPinterest}}
                 <a href="{{@config.custom.socialPinterest}}" aria-label="Pinterest">
+                    <desc>Pinterest</desc>
                     <svg>
                         <use xlink:href="{{@website.assetsUrl}}/svg/svg-map.svg#pinterest"/>
                     </svg>
                 </a>
             {{/if}}
             {{#if @config.custom.socialYoutube}}
                 <a href="{{@config.custom.socialYoutube}}" aria-label="Youtube">
```

Comment:
```suggestion
                <a href="{{@config.custom.socialYoutube}}" aria-label="YouTube">
```

Reply:
It's already four spaces indented, just like the rest. Is this a typo?